.github/workflows/benchmarking.yml

```diff
@@ -119,7 +119,16 @@ jobs:
       env:
         VIP_VAP_ACCESS_TOKEN: "${{ secrets.VIP_VAP_ACCESS_TOKEN }}"
       run: |
+        set -e
         cd zenith-perf-data
+        mkdir -p reports/
+        
+        echo "Generating report"
+        pipenv run python ../scripts/generate_perf_report_page.py --input-dir data/staging --out reports/staging.html 
+        
+        echo "Uploading perf result"
         git add data
+        git add reports
+
         git commit --author="vipvap <vipvap@zenith.tech>" -m "add performance test result for $GITHUB_SHA zenith revision"
         git push https://$VIP_VAP_ACCESS_TOKEN@github.com/zenithdb/zenith-perf-data.git master
```

Comment:
Would you consider extracting this code into a separate script? Otherwise, we'd have to make sure this piece is in sync with the one for CircleCI.